src/xrGame/Actor_Network.cpp

```diff
@@ -685,7 +685,7 @@ BOOL CActor::net_Spawn(CSE_Abstract* DC)
     inventory().SetPrevActiveSlot(NO_ACTIVE_SLOT);
 
     //-------------------------------------
-    m_States.empty();
+    m_States.clear(); // Xottab_DUTY: check if replace is correct
```

Comment:
Yeah, this looks correct, but should probably be tested if it doesn't have unintended consequences.